pkgs/by-name/fr/frr/package.nix

```diff
@@ -86,23 +85,15 @@ lib.warnIf (!(stdenv.buildPlatform.canExecute stdenv.hostPlatform))
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "frr";
-  version = "10.1";
+  version = "10.2";
```

Comment:
Fine, this could be merged together with https://github.com/NixOS/nixpkgs/pull/302815 (libyang-3 update).

Reply:
Done.